### DISCUSSION TITLE
Can't use any wrappers later than 2.9.0.7-rc1 on my intel hackintosh

### DISCUSSION THREAD

**Discussion by georgepenny** [The-Wineskin-Project/WineskinServer#206] [Q&A]
Hi,

I suspect that I have an 'unsupported' setup as I am running an intel Hackintosh on Mac OS Monterey but just wanted to post a question in case someone can help me or it helps someone else.

I have an issue where I can't use any master wrapper later than 2.9.0.7-rc1 on my intel Hackintosh as any wrapper (even blank ones) I create will not run as I get a Mac OS error stating that "You can’t open the application “test.app” because it may be damaged or incomplete".

I have tried a number of things to attempt to get this working and no luck so far:

- Uninstalling Wineskin completely, including removing ~/Library/Application Support/Wineskin
- Disabling SIP
- Enabling SIP
- Copying a wrapper generated from a MacBook Air directly to my Hackintosh
- Installing Wineskin via brew
- Installing Wineskin manually ensuring to remove the quarantine flag.

What is interesting, is that I copied a wrapper that was on 2.9.0.9-4 I made on my MacBook rather than Hackintosh and was able to run it off a usb memory stick on my Hackintosh. As soon as I copy it across to my Hackintosh ssd, I get the issue "You can’t open the application “test.app” because it may be damaged or incomplete".

Wondering if anyone has come across this issue or has any ideas for further debugging.

**Comment by Gcenx** [comment]
> What is interesting, is that I copied a wrapper that was on 2.9.0.9-4 I made on my MacBook rather than Hackintosh and was able to run it off a usb memory stick on my Hackintosh. As soon as I copy it across to my Hackintosh ssd, I get the issue "You can’t open the application “test.app” because it may be damaged or incomplete".

Something is defiantly screwy with your hackintosh setup but that’s not something I’m familiar with. I’d expect the wrapper to fail when run from the usb stick not just when copied to the internal SSD.
